kube/sixpack-web-controller.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: sixpack-web
+spec:
+  template:
+    metadata:
+      labels:
+        name: sixpack-web
+    spec:
+      containers:
+      - name: sixpack-web
+        image: leogamas/sixpack
+        command: ["sixpack.web:start"]
+        resources:
+          requests:
+            memory: "128M"
+            cpu: "0.2"
+          limits:
+            memory: "256M"
+            cpu: "0.5"
+        ports:
+        - containerPort: 5000:8000
+        env:
+        - name: SIXPACK_CONFIG_ENABLED
+          value: true
+        - name: SIXPACK_CONFIG_REDIS_PORT
+          value: 6379
+        - name: SIXPACK_CONFIG_REDIS_HOST
+          value: redis
```

Comment:
this name is right?  here we should use the name of redis service created by helm.